hoomd/data/local_access_gpu.py

```
@@ -5,10 +5,11 @@
 """Implement local access classes for the GPU."""
 
 from hoomd import _hoomd
+from hoomd.md import _md
```

What if we build with MD?

---

Do you mean what if we build WITHOUT md? That's a good point, and perhaps these class should sit in a `hoomd.md.data` namespace to avoid errors for hpmc only builds.

---

Yes that's what I meant. I couldn't remember if the flag was ENABLE_, BUILD_, etc., thanks for understanding my meaning. Edited for future clarity. 

And @b-butler @joaander are the authorities on the right way to do this. So probably let them weigh in on the correct location for this code to live before making the move.

---

HOOMD must import and pass tests when `BUILD_MD=off`. Do not use `try: import ... except: pass` constructions as they mask legitimate errors.

If possible, move code that needs to import `_md` to the `md` component. If not, make the non MD parts of this code work when `_md` is not available. Use the bool `hoomd.version.md_built` to see whether you can import `md` and `_md`.